Assets/Scenes/LevelOneScene.unity

```diff
@@ -330,7 +330,7 @@ Transform:
   m_ConstrainProportionsScale: 0
```

Comment:
Can you apply these changes to the level two scene as well?

Reply:
yep will do